main.js

```diff
@@ -19,49 +18,27 @@ Apify.main(async () => {
     if (!input) crash('Key-value store does not contain INPUT.');
     const { query } = input;
     if (!query || typeof query !== 'string') crash('Query loaded from INPUT is empty or not a string.');
-
-    const commonHeaders = {
-        'user-agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/69.0.3497.100 Safari/537.36',
-    };
-
     let attempts = 20;
     let response;
     do {
         try {
-            const sessionLength = 8;
-            const proxy = Apify.getApifyProxyUrl({
-                session: `ks${Math.floor(Math.random() * (10 ** sessionLength)).toString().padStart(sessionLength, '0')}`,
-            });
-
-            // Prepare cookie jar so that the second request contains cookies from the first one
-            const cookieJar = request.jar();
-            const preparedRequest = request.defaults({ jar: cookieJar, proxy });
-
-
-            // Query the url and load csrf token from it
-            const url = 'https://www.kickstarter.com/discover/advanced?result=all&term=game';
-            const html = await preparedRequest({
-                url,
-                headers: { ...commonHeaders },
-            });
-
-            const $ = cheerio.load(html);
-            const csrfToken = $('meta[name="csrf-token"]').attr('content');
-
-            if (!csrfToken) crash('Could not load CSRF token. Try again.');
-
-            // Query the kickstarter location search page with the correct csrf token and cookies
-            response = await preparedRequest({
+            const proxyConfiguration = await Apify.createProxyConfiguration();
+            // Query the kickstarter location search page
+            console.log(`making request to https://www.kickstarter.com/locations/search?searchable=true&term=${query}`);
+            const { body } = await Apify.utils.requestAsBrowser({
                 url: `https://www.kickstarter.com/locations/search?searchable=true&term=${query}`,
+                proxyUrl: proxyConfiguration.newUrl(),
                 headers: {
-                    accept: 'application/json, text/javascript, */*; q=0.01',
-                    'accept-language': 'en;q=0.9',
-                    referer: url,
-                    'x-csrf-token': csrfToken,
-                    'x-requested-with': 'XMLHttpRequest',
+                    // eslint-disable-next-line max-len
+                    'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.212 Safari/537.36',
```

Comment:
I think that using got-scraping internally by requestAsBrowser, passing the headers manually isn't the best way. only `Accept: 'application/json'` should remain